db/schema.rb

```diff
@@ -10,15 +10,19 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20170112024448) do
+ActiveRecord::Schema.define(version: 20170711171010) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
+  enable_extension "cube"
+  enable_extension "earthdistance"
 
   create_table "apartments", force: :cascade do |t|
     t.string   "street_address"
     t.datetime "created_at",     null: false
     t.datetime "updated_at",     null: false
+    t.float    "long"
+    t.float    "lat"
   end
```

Comment:
My curiosity got the better of me and I found this great discussion on precision/accuracy in coordinates-- looks like if we only need to be accurate in the realm of 10 meters (~30 feet), 6 decimal points is adequate. https://gis.stackexchange.com/questions/8650/measuring-accuracy-of-latitude-and-longitude